packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -3157,6 +3157,128 @@ describe.each([
               }
             )
           })
+
+          it("should be able to filter rows to be calculated on on the view", async () => {
```

Comment:
This description sounds a little weird. Is this correct?

Reply:
It's _technically_ correct grammar but it is a very odd sentence. I'll reword it.